migrations/v3.js

```diff
@@ -0,0 +1,26 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-assessment - v2.2.0 > v3.0.0', async () => {
+  let course, assessmentConfig;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.2.0', { name: 'adapt-contrib-assessment', version: '<3.0.0' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    course = content.filter(({ _type }) => _type === 'course');
+    assessmentConfig = course.find(({ _assessment }) => _assessment);
+    if (assessmentConfig) return true;
+  });
```

Comment:
Can simplify now with our new helpers (will need to import)

```suggestion
  whereContent('adapt-contrib-assessment - where assessment', async content => {
    assessmentConfig = getCourse().find(({ _assessment }) => _assessment);
    if (assessmentConfig) return true;
  });
```